QUARKUS-4430.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-3535 - Add a metric for rejected submissions to the pool in RHBQ 3.2
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4430
+
+PR: https://github.com/quarkusio/quarkus/pull/39583
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/39581
+
+Verify that URIs in metrics tags are correctly recorded when they match a client or server pattern,
+especially in cases of redirection (302) and not found (404) responses.
+
+## Scope of the testing
+* Run Quarkus micrometer Prometheus application with OpenId Connect Keycloak and verify in case of redirections (302)
+  and not found (404) response the Uris metrics tags are correctly recorded.
+* The test coverage will be verified for JVM.
+
+
+## Existing test coverage
+* We have already tests related to monitoring metrics with Prometheus and OpenId Connect Keycloak but not specific test
+  about this improvement feature related to tag redirections and not found responses.
+ 
+
+### Impact on test suites and testing automation
+* New test class will be added into [monitoring](https://github.com/quarkus-qe/quarkus-test-suite/tree/main/monitoring/micrometer-prometheus-oidc) 
+
+### Impact on resources
+* One baremetal test execution in JVM mode
```

Comment:
we generally test everything in both JVM and native mode unless we have a reason for the opposite; now, the Keycloak only runs in JVM IIRC but we don't care about Keycloak per say, just about Quarkus